oap-server/server-storage-plugin/storage-elasticsearch-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/elasticsearch/base/StorageEsInstaller.java

```diff
@@ -148,7 +149,7 @@ private void createTimeSeriesTable(Model model) throws StorageException {
             if (shouldUpdateTemplate) {
                 structures.putStructure(tableName, mapping);
                 boolean isAcknowledged = esClient.createOrUpdateTemplate(
-                    tableName, settings, structures.getMapping(tableName), config.getIndexTemplateOrder());
+                    tableName, settings, mapping, config.getIndexTemplateOrder());
```

Comment:
I'm not sure why use `structures.getMapping(tableName)` before. In that logic, the updated template would only add new mapping property but wouldn't update/delete the existing property.